src/main/resources/templates/openapi/templateSchemaWithLombok.ftlh

```diff
@@ -61,4 +65,36 @@ public class ${schema.className} {
   </#if>
 
 </#list>
+
+  @Builder
+  private ${schema.className}(<@compress single_line=true><#list schema.fieldObjectList as field>
+    <#if field.required>@NonNull </#if><#if field.dataType?has_content && field.dataTypeSimple == "enum">${field.baseName?cap_first} ${field.baseName}<#elseif field.dataType?has_content && field.dataTypeSimple == "array">List<${field.dataType}> ${field.baseName}<#elseif field.dataType?has_content && field.dataTypeSimple == "map">Map<String,${field.dataType}> ${field.baseName}<#else>${field.dataTypeSimple?cap_first} ${field.baseName}</#if><#sep>, </#list>){</@compress>
+    <#list schema.fieldObjectList as field>
+    this.${field.baseName} = ${field.baseName};
+    </#list>
+
+    <#if schema.schemaCombinator == "anyOf" || schema.schemaCombinator == "oneOf">
+    validatePartialCombinations();
+    </#if>
+  }
+
+  <#if schema.schemaCombinator == "anyOf" || schema.schemaCombinator == "oneOf">
+  private void validatePartialCombinations() {
+    boolean satisfiedCondition = false;
+
+  <#list schema.fieldObjectList as field>
+    <#if field == schema.fieldObjectList?first>
+    if (Objects.nonNull(this.${field.baseName})) {
+    <#else>
+    else if (Objects.nonNull(this.${field.baseName})) {
```

Comment:
Shouldn't there be a key before de 'else if' to close the 'if' statement?

Reply:
Both the if and the else if are closed at line 92. Whether it is an if or an else if depends on if it is the first field that is being processed.

Reply:
I see, thanks!